packages/compass-e2e-tests/helpers/compass.ts

```diff
@@ -787,7 +792,43 @@ export async function startBrowser(
     ...webdriverOptions,
     ...wdioOptions,
   })) as CompassBrowser;
-  await browser.navigateTo('http://localhost:7777/');
+
+  if (ATLAS_DOMAIN) {
+    // Navigate to a 404 page to set cookies
+    await browser.navigateTo(`https://${ATLAS_DOMAIN}/404`);
+
+    const cookiesFile = process.env.COOKIES_FILE;
+    if (!cookiesFile) {
+      throw new Error(
+        'ATLAS_DOMAIN is set but COOKIES_FILE is not. Please set COOKIES_FILE to the path of the cookies file.'
+      );
+    }
+    const cookies = JSON.parse(await fs.readFile(cookiesFile, 'utf8'));
```

Comment:
Can you add some type definitions for this file?

Reply:
You mean like, for the cookies object?

Reply:
Yes, something like:

```suggestion
    type StoredAtlasCloudCookies = {...some shape}[]
    const cookies: StoredAtlasCloudCookies = JSON.parse(await fs.readFile(cookiesFile, 'utf8'));
```

We probably don't need any strict validation, but without the type it's unclear what's the expected shape for the content of the COOKIES_FILE file and taking into account that it will be generated in a completely separate repo, it's probably our best way of documenting this